.github/workflows/run-regression-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Test VSCode extension
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        runtime: [ linux, mac ]
+     # Removing windows as a standard run flavor   runtime: [ linux, mac, windows ]
+        targetVSCode: [latest ]
+      # As Vscode 1.78 is the only support version for current release, running test with only latest
+      #Enable it in future for multiple releases as applicable  targetVSCode: [ previousMinusOne, previous, latest ]
```

Comment:
VS Code just released a new version (1.79) yesterday - so can the matrix be updated to test latest and previous? (1.79 and 1.78)

Reply:
Can this be taken as a seperate item post GA  for the new team? Latest always picks the latest supported release

Reply:
Enabled latest, previous which should take care of this requirement

Reply:
Addressed in the latest commit, Pls check